app/src/main/java/edu/uc/groupProject/topten/ui/main/MainFragment.kt

```diff
@@ -19,13 +18,10 @@ import edu.uc.groupProject.topten.R
  * MainFragment class.
  *
  * Responsible for creating the view, and additionally contains Observe functionality that
- * connects to the database.
```

Comment:
Fragments should also be getting data from a ViewModel.

Reply:
Yes